### ISSUE TITLE
List profile issues - IG publisher failing to complete

### ISSUE THREAD

**Issue by robeastwood-agency** [AuDigitalHealth/ci-fhir-stu3#23]
Two issues are being experienced related to the List resource:
- within a bundle instance, linking a List entry to an associated composition entry
- adding a meta.profile value within a List resource

### Expected behaviour
In both of the above circumstances, the IG publisher should complete and the content be successfully generated

### Actual behaviour
The IG publisher fails to complete with a non-helpful error message of:
```
Exception in thread "main" java.lang.Error: Not done yet (ValidatorHostServices.resolveFunction): source
```
in the following circumstances:

1. When `Bundle.entry.resource.List` is referenced by its exact `fullUrl` value from a corresponding `Composition.section.entry.reference` element
2. When a List example, either stand-alone or within a bundle, has a meta.profile reference to the Agency profile "http://ns.electronichealth.net.au/ci/fhir/3.0/StructureDefinition/list-medseventchanges-1"

### Steps to reproduce

* Replicate either of the above 2 circumstances

### Environment

N/A

**Comment by dtr-agency** [comment]
@robeastwood-agency I think this is an issue with the profile, not the base List resource.

**Comment by dtr-agency** [comment]
Sorry, slip of the mouse.

**Comment by richardtownleyoneill-ADHA** [comment]
Issue resolved in https://jira.digitalhealth.gov.au/browse/CIFMM-2378
https://stash.digitalhealth.gov.au/projects/CIL/repos/au-fhir-base-fork/commits/a6fc9c6127a8a2b3ab6d9ea22e879ab06785ade5 
The HL7 AU profile http://hl7.org.au/fhir/StructureDefinition/au-medlist invariant inv-lst-0 had invalid FHIRPath

**Comment by dtr-agency** [comment]
partial fix via hl7au/au-fhir-base#262

**Comment by dtr-agency** [comment]
To be fixed via https://jira.digitalhealth.gov.au/browse/CIFMM-2390
